app/api/webhooks/db-record-created/route.ts

```
@@ -0,0 +1,53 @@
+import { NextResponse } from "next/server";
+import { createClient } from "@/supabase/server";
```

ESLint

[error] 'createClient' is defined but never used.

(@typescript-eslint/no-unused-vars)